src/main/java/ticTacToe/player/SmartComputer.java

```diff
@@ -89,4 +90,12 @@ private void applyMiniMax(Map<Integer, Grid> gridsWithScores, String currentMark
             score = Collections.min(gridsWithScores.keySet());
         }
     }
+
+    private void slowDown() {
+        try {
+            Thread.sleep(1200);
```

Comment:
Well done for figuring out how to make it slower! But the issue is now that your tests are very slow... 
Do you know why slow tests are a problem and how to rectify this problem?

Reply:
Because there might be hundreds of tests to run and it could last for ever? 🤔 
I have no idea how to rectify this but I'll try to figure it out!

Reply:
Yep! And the problem is that humans are lazy and don't like waiting... so the slower the test suite is the less we run it and it's just all a whole big downward spiral from there...